### DISCUSSION TITLE
Inverting single Click/Double Click?

### DISCUSSION THREAD

**Discussion by antoniograzioli** [probonopd/MiniDexed#300] [Ideas]
I find the actual Double Click (for "OK") and Singe Clik (for "Back") command somewhat counterintuitive and misleading, 
Is there -or is it possible- to add some kind of configuration option to invert them?
I think that one single click should be used to confrim choices and a dblclick to go back one step...
is it just me?

**Comment by antoniograzioli** [comment]
it looks like it should only be a matter of inverting these lines of code in userinterface.cpp (or adding a parameter to them to handle click/dblclick):

```

       case CKY040::EventSwitchClick:
		m_Menu.EventHandler (CUIMenu::MenuEventBack);
		break;

	case CKY040::EventSwitchDoubleClick:
		m_Menu.EventHandler (CUIMenu::MenuEventSelect);
		break;

	case CKY040::EventSwitchTripleClick:
		m_Menu.EventHandler (CUIMenu::MenuEventHome);
		break;

	case CKY040::EventSwitchHold:
		if (m_pRotaryEncoder->GetHoldSeconds () >= 120)
		{
			delete m_pLCD;		// reset LCD

			reboot ();
		}
		break;
```

**Reply by stevelittlefish** [reply]
This functionality is hopefully coming soon.  I've already done the coding and it's waiting for code review to be merged.

See: https://github.com/probonopd/MiniDexed/pull/274
